app/services/ersTemplatesInfo/OtherTemplateInfo.scala

```diff
@@ -33,7 +33,8 @@ trait OtherTemplateInfo {
 
   val otherSheet1Desc = "Other ERS schemes or arrangements – Grant of securities options, including Restricted Stock Units (RSUs)"
   val otherSheet1ValConfig = "ers-other-grants-validation-config"
-  val otherSheet2Desc = "Other ERS schemes or arrangements – Other option events, including conditional awards.  Options lapsed for nil consideration should not be included"
+  val otherSheet2Desc: String = "Other ERS schemes or arrangements – Other option events, including conditional awards.  " +
```

Comment:
Was there a reason for giving just this one a type definition?

Reply:
None. I don't know why I did that, my brain my have been on autopilot and I was probably going to add the return type for all of the strings but decided against it.